pkg/config/doc_test.go

```diff
@@ -0,0 +1,39 @@
+package config_test
+
+import (
+	"fmt"
+
+	"github.com/dcos/dcos-cli/pkg/config"
+)
+
+// Create a config from a TOML file path.
```

Comment:
s/`Create a config`/`Creates a Config`

Reply:
I've changed the name of the example functions, now they appear right after the doc for their associated functions, I think it's more readable like this and doesn't need a comment, so I've removed it and the one for FromString too.